src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <app-category-edit
+      :category="targetCategoryName"
+      :error-message="errorMessage"
+      :done-message="doneMessage"
+      :is-loading="isLoading"
+      @update-value="updateValue"
+      @handle-submit="updateTargetCategoryName"
+    />
+  </div>
+</template>
+
+<script>
+import { CategoryEdit } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    errorMessage() {
+      return this.$store.state.categories.errorMessage;
+    },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
+    targetCategoryName() {
+      return this.$store.state.categories.targetCategory.name;
+    },
+    isLoading() {
+      return this.$store.state.categories.isLoading;
+    },
+  },
+  created() {
+    this.$store.dispatch('categories/getCategoryDetails', this.$route.params.id);
+  },
+  destroyed() {
+    this.$store.dispatch('categories/inputTargetCategoryName', '');
+  },
+  methods: {
+    updateValue(event) {
+      this.$store.dispatch(
+        'categories/inputTargetCategoryName',
+        event.target.value,
+      );
+    },
+    updateTargetCategoryName() {
+      this.$store.dispatch('categories/updateTargetCategoryName');
+    },
```

Comment:
> "updateValue"と"inputTargetCategoryName"で、
> 記述に相違があることに違和感を感じています。

なるほど！
確かにここに関しては気になるかもしれないです。

以下のような記述が一番わかりやすいかなと思うのですが、いかがでしょう。

```suggestion
    updateTargetCategoryName(event) {
      this.$store.dispatch(
        'categories/updateTargetCategoryName',
        event.target.value,
      );
    },
    handleSubmit() {
      this.$store.dispatch('categories/updateCategory');
    },
```

- updateTargetCategoryName(event)
  - 入力中の文字で更新対象のカテゴリ名を更新していく
- handleSubmit()
  - 更新が確定して送信する
- this.$store.dispatch('categories/updateCategory')
  - 〇〇Category（〇〇は CRUD などの動詞）で API を実行する actions というように統一する。